velox/docs/functions/spark/string.rst

```diff
@@ -23,6 +23,21 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
         SELECT contains('Spark SQL', null); -- NULL
         SELECT contains(x'537061726b2053514c', x'537061726b'); -- true
 
+.. spark:function:: conv(number, fromBase, toBase) -> varchar
+
+    Converts ``number`` represented as a string from ``fromBase`` to ``toBase``.
+    ``fromBase`` must be a value inclusively between 2 and 36. ``toBase`` must be a value
+    inclusively between 2 and 36 or between -36 and -2. Negative ``toBase`` means output
```

Comment:
> Negative ``toBase`` means output
>     will be a signed number and positive ``toBase`` means it will be an unsigned number.

I don't understand this. Would you clarify?

For example, what's the difference between conv('100', 2, 10) and conv('100', 2, -10) or conv('-100', 2, 10) and conv('-100', 2, -10).

Reply:
Hi @mbasmanova, the result:
```
+----------------+-----------------+--------------------+------------------+
|conv(100, 2, 10)|conv(100, 2, -10)|   conv(-100, 2, 10)|conv(-100, 2, -10)|
+----------------+-----------------+--------------------+------------------+
|               4|                4|18446744073709551612|                -4|
+----------------+-----------------+--------------------+------------------+
```
For conv(-100, 2, 10), since toBase is positive, returns unsigned result, i.e., kMaxInt64  + (-4)  + 1.